src/idl/include/idl/tree.h

```diff
@@ -353,6 +365,8 @@ struct idl_case {
   idl_case_label_t *labels;
   idl_type_spec_t *type_spec;
   idl_declarator_t *declarator;
+  idl_id_t id;
```

Comment:
@reicheratwork added the `id` field to the declarator in #963, probably best to include unions there and make this work with that(?)

Reply:
Yes, I'll remove the id field here after rebasing on master with that PR included.